website/templates/profile/notifications.mako

```diff
@@ -25,10 +25,19 @@
                         <div class="form-group">
 
                             <input type="checkbox"
-                                   data-bind="checked: subscribed"/>
-                            <label data-bind="text: list"></label>
+                                   data-bind="checked: subscribed" value="Open Science Framework General"/>
+                            <label data-bind="text: list[0]"></label>
```

Comment:
Can you look into refactoring the view model so that you don't need to rely on the hard-coded indices?